hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/graphite/WebTargetGraphiteClient.java

```diff
@@ -25,8 +25,8 @@
     public WebTargetGraphiteClient(WebTarget webTarget) {
         this.webTarget = webTarget
                 .path("render")
-                .queryParam("from", "-1minutes")
-                .queryParam("until", "now")
+                .queryParam("from", "-5minutes")
```

Comment:
Do we need 3 minutes of data? The logic fetches just the last data point. I just want to confirm whether graphite guarantees to return just the data points which are available for this query.